sys/netbsd/sys.txt

```diff
@@ -36,8 +36,11 @@ compat_43_fstat43(fd fd, sb ptr[out, stat43])
 __fstat50(fd fd, sb ptr[out, stat])
 compat_20_statfs(path ptr[in, filename], buf ptr[out, statfs12])
 compat_20_fstatfs(fd fd, buf ptr[out, statfs12])
+fstatat(fd fd, path ptr[in, filename], buf ptr[out, stat], flag flags[fstatat_flags])
```

Comment:
Can we specify somehow `AT_FDCWD` support?

Reply:
fd is defined as:
https://github.com/google/syzkaller/blob/cb93dc6ac64225e09f44bac6c6cce1dae1b248b0/sys/netbsd/fs.txt#L13